test/functional/tests/volumes/test_many_cores_on_many_lvms.py

```diff
@@ -0,0 +1,94 @@
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+import pytest
+from storage_devices.lvm import Lvm, LvmConfiguration
+from tests.common.lvm_common_utils import run_fio_on_lvm, get_test_configuration
+from api.cas import casadm
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_utils.size import Size, Unit
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_many_cores_on_many_lvms():
+    """
+        title: Test for CAS creation with lvms as cores: 1 cache, 16 lvms, 16 cores.
+        description: |
+          Validation of LVM support, CAS with 1 cache and 16 lvms as 16 cores.
+        pass_criteria:
+          - LVMs created successfully.
+          - CAS devices created successfully.
+          - FIO with verification ran successfully.
+          - Configuration after reboot match configuration before.
+    """
+    with TestRun.step(f"Prepare devices."):
+        cache_device = TestRun.disks['cache']
+        core_device = TestRun.disks['core']
+
+        cache_device.create_partitions([Size(10, Unit.GibiByte)])
+        core_device.create_partitions([Size(10, Unit.GibiByte)])
+
+        cache_dev = cache_device.partitions[0]
+        core_dev = core_device.partitions
```

Comment:
It's intentional, this should be a list and be passed to "create_specific_lvm_configuration" like that - tested out:)

Reply:
This is clever, but not easy to understand immediately without an explanation.
Imo doing `.partitions[0]` and then passing `[core_dev]` to lvm config would make it easier to understand both: why there is a missing partition index (mostly because there will be no missing index 😄), and what is the meaning of this argument to `Lvm.create_specific_lvm_configuration()`.